edx/edx-ora2
# [BD-05] [TNL-7310] [BB-3388] Pluggable ORA Submission Editor - TinyMCE

Depends on https://github.com/edx/edx-ora2/pull/1574

This PR adds TinyMCE v5 as the WYSIWYG editor for ORA Response.

**JIRA tickets**:
- https://openedx.atlassian.net/browse/TNL-7310
- https://tasks.opencraft.com/browse/BB-3388

**Discussions**: None

**Dependencies**: https://github.com/edx/edx-ora2/pull/1574

**Screenshots**: 
![Screenshot from 2021-01-22 10-34-42](https://user-images.githubusercontent.com/1010244/105447832-32fe1580-5c9f-11eb-917d-07ff6f4e55cf.png)


**Sandbox URL**: https://bb3388.sandbox.opencraft.hosting/

**Merge deadline**: None

**Testing instructions**:

1. Clone this PR on your local devstack. Install using make install-local-ora.
2. Create an ORA block. On the settings tab select ``WYSIWYG Editor`` from Response Editor Dropdown.
3. Publish ORA Block
4. Check that TinyMCE editor loaded in both LMS & Studio. 
5. Write on the editor, check ``Save Your Progress`` becomes enabled.
6. Click on ``Save Your Progress`` and reload the page. You should come back where you left off.
7. Check if the submission process working.

**Author notes and concerns**:

1. This PR includes TinyMCE v5 and all of the assets related to the editor.

**Reviewers**
